### DISCUSSION TITLE
Vanara is on stage of Microsoft Hack Together-Contest

### DISCUSSION THREAD

**Discussion by tajbender** [dahall/Vanara#379] [Show and tell]
Good Morning, Sir,

just wanted to let you know that electrifier will use Vanara in a contest:

[Microsoft Hack Together](https://github.com/microsoft/hack-together/issues/38)

I'f you'd like to know more please don't hesitate to contact me.

Thx and regards,

tajbender

**Comment by dahall** [comment]
Exciting. Best of luck!

**Reply by tajbender** [reply]
Thx 💭

**Comment by tajbender** [comment]
### fyi

https://github.com/microsoft/hack-together/issues/38

https://github.com/electrifier/electrifier

Good luck 2 us 🥠

**Comment by tajbender** [comment]
I did miss the end date by three hours cause of an easy build problem 😂
